google-cloud-storage/src/main/java/com/google/cloud/storage/MultipartUploadClient.java

```diff
@@ -60,6 +63,18 @@ public abstract CreateMultipartUploadResponse createMultipartUpload(
   @BetaApi
   public abstract ListPartsResponse listParts(ListPartsRequest listPartsRequest) throws IOException;
 
+  /**
+   * Aborts a multipart upload.
+   *
+   * @param request The request object containing the details for aborting the multipart upload.
+   * @return An {@link AbortMultipartUploadResponse} object.
+   * @throws IOException if an I/O error occurs.
+   * @throws NoSuchAlgorithmException if the specified algorithm is not available.
+   */
+  @BetaApi
+  public abstract AbortMultipartUploadResponse abortMultipartUpload(
+      AbortMultipartUploadRequest request) throws IOException, NoSuchAlgorithmException;
```

Comment:
We should never throw a `NoSuchAlgorithmException`. What necessitates this being declared here?

Reply:
Done